package.json

```diff
@@ -31,6 +31,8 @@
     "templates/"
   ],
   "dependencies": {
+    "chalk": "^1.1.1",
```

Comment:
Can we use gutil.colors instead of chalk? So we don't need to depend on another package.